nineslice.d.ts

```diff
@@ -116,6 +116,9 @@ declare namespace Phaser.GameObjects {
          */
         nineslice(x: number, y: number, width: number, height: number, key: string,
             nonUniform: number[]): Phaser.GameObjects.RenderTexture;
+
+        nineslice(x: number, y: number, width: number, height: number, { key: string, frame: any },
```

Comment:
Without this method definition, the compiler shows an error when trying to pass the 5th parameter as a json when working on typescript